README.md

```diff
@@ -1,5 +1,7 @@
 # zkNotary: a zkOracle for Mina powered by TLSNotary
 
+**Note to Builders**: We have written an article explaning, in technical terms, the most relevant parts of our code, the current state of the project and what needs to be done to push it forward. The write up can be found here: https://hackmd.io/@racampos/zknotary_status. 
```

Comment:
To limit the amount of `Note ...:` in the beginning and highlight the writeup, we could just center this part in the read me and make the entire sentence bold. So maybe just take out the **Note to builders part**, center the text, make it all bold, and add three dashes below it?